Floofbot/Modules/Tag.cs

```diff
@@ -7,191 +7,188 @@
+
+            while (result.Read()) {
+                tags.Add(result.GetString(0).Split(':')[0]);
+            }
 
-                tags = tags.OrderBy(x => x).ToList();
-                int index = 0;
-                for (int i = 1; i <= (tags.Count / 50) + 1; i++) {
-                    string text = "```glsl\n";
-                    int pagebreak = index;
-                    for (; index < pagebreak + 50; index++) {
-                        if (index < tags.Count) {
-                            text += $"[{index}] - {tags[index].ToLower()}\n";
-                        }
+            tags = tags.OrderBy(x => x).ToList();
+            int index = 0;
+            for (int i = 1; i <= (tags.Count / 50) + 1; i++) {
+                string text = "```glsl\n";
+                int pagebreak = index;
+                for (; index < pagebreak + 50; index++) {
+                    if (index < tags.Count) {
+                        text += $"[{index}] - {tags[index].ToLower()}\n";
                     }
+                }
 
-                    text += "\n```";
-                    pages.Add(text);
-                };
+                text += "\n```";
+                pages.Add(text);
+            };
 
-                // await PagedReplyAsync(pages);
-            }
+            // await PagedReplyAsync(pages);
+        }
 
-            [Command("remove")]
-            [RequireUserPermission(GuildPermission.ManageMessages)]
-            public async Task Remove(string tag)
-            {
-                string tagId = $"{tag.ToString()}:{Context.Guild.Id.ToString()}";
+        [Command("remove")]
+        [RequireUserPermission(GuildPermission.ManageMessages)]
+        public async Task Remove(string tag)
+        {
+            string tagId = $"{tag.ToString()}:{Context.Guild.Id.ToString()}";
 
-                string select = @"SELECT COUNT(*) FROM Tags WHERE TagID = $TagId";
-                SqliteCommand command = new SqliteCommand(select, dbConnection);
-                command.Parameters.Add(new SqliteParameter("$TagId", tagId));
+            string select = @"SELECT COUNT(*) FROM Tags WHERE TagID = $TagId";
+            SqliteCommand command = new SqliteCommand(select, dbConnection);
+            command.Parameters.Add(new SqliteParameter("$TagId", tagId));
 
-                if (Convert.ToInt32(command.ExecuteScalar()) > 0) {
-                    string delete = @"DELETE FROM Tags WHERE TagID = $TagId";
-                    command = new SqliteCommand(delete, dbConnection);
-                    command.Parameters.Add(new SqliteParameter("$TagId", tagId));
-                    command.ExecuteScalar();
+            if (Convert.ToInt32(command.ExecuteScalar()) > 0) {
+                string delete = @"DELETE FROM Tags WHERE TagID = $TagId";
+                command = new SqliteCommand(delete, dbConnection);
+                command.Parameters.Add(new SqliteParameter("$TagId", tagId));
+                command.ExecuteScalar();
```

Comment:
Do you really need to count the rows here?

Reply:
I'd prefer handling this in a future PR that deals with the models and/or the entity framework integration.

Reply:
Actually, yes. I tried logging the output of the result of the delete query and it was null.